core/src/main/kotlin/org/jetbrains/research/testspark/core/utils/CommandLineRunner.kt

```diff
@@ -9,13 +9,13 @@ class CommandLineRunner {
         protected val log = KotlinLogging.logger {}
 
         /**
-         * Executes a command line process and returns the output as a string.
+         * Executes a command line process
          *
          * @param cmd The command line arguments as an ArrayList of strings.
-         * @return The output of the command line process as a string.
+         * @return A pair containing exit value and a string message containing execution results
          */
-        fun run(cmd: ArrayList<String>): String {
-            var errorMessage = ""
+        fun run(cmd: ArrayList<String>): Pair<Int, String> {
```

Comment:
Instead of returning a pair, return a data class. 

Now, as a reviewer I cannot understand what `first` and what `second` actually are. 

In practice, a data class should always be preferred over a pair, especially in our case where a lot of components depend on it.

Reply:
I've done it to match the current style of the `compileCode` method. I do find it kinda ugly, so I will refactor both signatures

Reply:
>  I will refactor both signatures

Yeah, that is even better. Thanks!